front/public/locales/en/stdcm.json

```
@@ -127,6 +128,12 @@
   "stdcmErrors": {
     "bothPointAreScheduled": "The calculation cannot take into account both an origin and a destination schedule. You have to choose one or the other:",
     "infraNotLoaded": "Infrastructure is loading",
+    "invalidInformations": "Invalid information for calculation:",
+    "invalidFields": {
+      "totalMass": "the total mass of the consist is invalid",
+      "totalLength": "the total length of the consist is invalid",
+      "maxSpeed": "the maximum speed of the consist is invalid"
```

These values are in a section `Invalid information for calculation`, is it necessary to repeat that they are invalids ? (same in french)
```suggestion
      "totalMass": "the total mass",
      "totalLength": "the total length",
      "maxSpeed": "the maximum speed"
```

---

done